soundlib/Load_stm.cpp

```diff
@@ -80,8 +80,7 @@ static bool ValidateHeader(const STMFileHeader &fileHeader)
 		|| (fileHeader.dosEof != 0x1A && fileHeader.dosEof != 2)	// ST2 ignores this, ST3 doesn't. putup10.stm / putup11.stm have dosEof = 2.
 		|| fileHeader.verMajor != 2
 		|| (fileHeader.verMinor != 0 && fileHeader.verMinor != 10 && fileHeader.verMinor != 20 && fileHeader.verMinor != 21)
-		|| fileHeader.numPatterns > 64
-		|| fileHeader.globalVolume > 64)
+		|| fileHeader.numPatterns > 64)
```

Comment:
Why remove to globalVolume check all together? Couldn't this be made conditional on the file format version, which is already known at this point.

Reply:
Or, if the value in such files is expected to be zero, we do not need to remove it either. As I explained in your pull request for XMP, rejecting nonsensical values that cannot be found in real-world files is crucial for libraries when dealing with formats like STM that do not have many sensible "magic bytes".

Reply:
Sorry, I just noticed the bit about the default 0x58 value that might be used.

Reply:
As a result, I would rather replace it with this:
`(fileHeader.globalVolume > 64 && fileHeader.globalVolume != 0x58)`